pkg/config/runtime_config.go

```diff
@@ -82,15 +87,33 @@ func BuildRestConfig(rtCfg RuntimeConfig) (*rest.Config, error) {
 	return restCFG, nil
 }
 
+// BuildCacheOptions returns a cache.Options struct for this controller.
+func BuildCacheOptions() cache.Options {
+	cacheOptions := cache.Options{
+		ReaderFailOnMissingInformer: true,
+		ByObject: map[client.Object]cache.ByObject{
+			&corev1.Pod{}: {
+				Transform: k8s.StripDownPodTransformFunc,
+			},
+			&corev1.Service{}: {
+				Transform: k8s.StripDownServiceTransformFunc,
+			},
+			&corev1.Namespace{}:           {},
+			&networkingv1.NetworkPolicy{}: {},
```

Comment:
Do we need explicitly cache on PolicyEndpoints as well?

Reply:
yes, I will add it